TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmode/teleop/TeleOpMain.java

```diff
@@ -67,11 +62,11 @@ public void runOpMode() {
             }
 
             if (gamepad1.dpad_up) {
-                robot.armLeft.scanServoAsync(robot.armLeft.getPosition() + 0.01, 20);
-                robot.armRight.scanServoAsync(robot.armRight.getPosition() + 0.01, 20);
+                robot.armLeft.setPosition(Range.clip(robot.armLeft.getPosition() - 0.001, 0.6, 1));
```

Comment:
Look at the feedback on the SmartServo class. TL;DR: we need to make a non-blocking function that checks if enough time has elapsed so we can control the servos better.

Reply:
I haven't seen an issue with this specific to the arm, there have only been issues with the claw. Is this something that needs to change?

Reply:
The speed of the arm is dependent on what else the robot is doing in that loop. By switching to the new method, it should get rid of a lot of invariability.